<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GGCreateSceneDlg</class>
 <widget class="QDialog" name="GGCreateSceneDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Scene name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtSceneName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Scene media directory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtSceneDir">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GGCreateSceneDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GGCreateSceneDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtSceneName</sender>
   <signal>textEdited(QString)</signal>
   <receiver>GGCreateSceneDlg</receiver>
   <slot>checkOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtSceneDir</sender>
   <signal>textEdited(QString)</signal>
   <receiver>GGCreateSceneDlg</receiver>
   <slot>checkOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkOk()</slot>
 </slots>
</ui>
